etc/tests/test_integration.py

```diff
@@ -117,7 +117,8 @@ def get_katsu_datasets(user):
     return response.json()["result"]
 
 
-def add_program_authorization(dataset):
+def add_program_authorization(dataset: str, curators: list, 
```

Comment:
@SonQBChau , I removed adding site admin as curator and team member by default here

Reply:
LGTM